design-documents/graph-ql/coverage/PaginationChangesForEntities.graphqls

```diff
@@ -0,0 +1,25 @@
+type Cart {
+    items(
```

Comment:
If we need field type change, a new field with version suffix should be introduced, `items_v2` in this case.

Reply:
The same is related to other entities in this list.

Reply:
should we deprecate the field?

Reply:
Yes, let's deprecate the old fields.